drivers/serial/uart_nrfx_uarte.c

```diff
@@ -168,20 +179,25 @@ struct uarte_nrfx_data {
 	struct uarte_async_cb *async;
 #endif
 	atomic_val_t poll_out_lock;
-	uint8_t *char_out;
-	uint8_t *rx_data;
+#ifdef UARTE_ENHANCED_POLL_OUT
 	uint8_t ppi_ch_endtx;
+#endif
+	atomic_t flags;
```

Comment:
nit: place it before `ppi_ch_endtx` to avoid padding.

Reply:
Done